api/tests/prescription/learner-management/integration/infrastructure/repositories/organization-learner-repository_test.js

```diff
@@ -929,4 +933,142 @@ describe('Integration | Repository | Organization Learner Management | Organizat
       });
     });
   });
+
+  describe('#saveCommonOrganizationLearners', function () {
+    let organizationId;
+    beforeEach(async function () {
+      organizationId = databaseBuilder.factory.buildOrganization().id;
+      await databaseBuilder.commit();
+    });
+
+    context('add new learner', function () {
+      it('should save the new learner', async function () {
+        const learnerData = new CommonOrganizationLearner({
+          firstName: 'Sacha',
+          lastName: 'Du Bourg Pallette',
+          organizationId,
+          attributes: {
+            firstName: 'Sacha',
+            lastName: 'Du Bourg Pallette',
```

Comment:
```suggestion
          lastName: 'Du Bourg Palette',
          organizationId,
          attributes: {
            firstName: 'Sacha',
            lastName: 'Du Bourg Palette',
```
On rigole pas avec Pokémon :sob:

Reply:
ok la police de pokémon est aux aguets, je vais corriger ça !